app/views/schemes/index.html.erb

```diff
@@ -6,7 +6,9 @@
 <%= render partial: "organisations/headings", locals: current_user.support? ? { main: "Supported housing schemes", sub: nil } : { main: "Supported housing schemes", sub: current_user.organisation.name } %>
 
 <% if SchemePolicy.new(current_user, nil).create? %>
-  <%= govuk_button_link_to "Create a new supported housing scheme", new_scheme_path, html: { method: :post } %>
+  <div class="govuk-button-group govuk-!-margin-bottom-6">
+    <%= govuk_button_link_to "Create a new supported housing scheme", new_scheme_path, html: { method: :post } %>
+  </div>
```

Comment:
Added a bottom margin to this button to align content vertically with other pages (lettings log and sales log). Included the button group, like in those pages, should there be any future additions.